CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added support for `ets:update_counter/3` and `ets:update_counter/4`.
 - Added `erlang:+/1`
 - Added `lists:append/1` and `lists:append/2`
+- Share `erlang:binary_part/3` as BIF and NIF
```

Comment:
This can be removed: we add to CHANGELOG only user visible changes, such as fixing a bug or adding a new function. Changes such as refactoring and similar ones are not added to the changelog.